docs/Windows-README.md

```diff
@@ -18,47 +18,51 @@ available from Microsoft.com (search for "visual c 2008 redistributable").
 Alternatively, you can install the redistributable in a "side-by-side"
 configuration, which does not require administrator privileges. Simply
 download the DLL package from:
-https://download.savannah.gnu.org/releases/rdiff-backup/Microsoft.VC90.zip
+[https://download.savannah.gnu.org/releases/rdiff-backup/Microsoft.VC90.zip](https://download.savannah.gnu.org/releases/rdiff-backup/Microsoft.VC90.zip)
 and copy the four enclosed files to the same directory as rdiff-backup.exe.
 
 You will need to follow either method only once.
 
 
 ADDITIONAL ISSUES:
```

Comment:
Put ## in front of each header, remove column at the end, put title case

Reply:
I've actually made them a H3, as H2 still looked too big in HTML.

Reply:
Okay, dropped back to H2, just to be consistent.